src/presentation/controllers/WebhookController.ts

```
@@ -20,6 +24,10 @@ export class WebhookController {
           const subscription = event.data.object;
           await this.updateSubscriptionStatusUseCase.execute(subscription);
           break;
+        case "checkout.session.completed":
+          const session = event.data.object;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)